convene-web/app/controllers/rooms_controller.rb

```diff
@@ -1,5 +1,9 @@
 class RoomsController < ApplicationController
   def show
-    @room = current_workspace.rooms.friendly.find(params[:id])
+    current_room
   end
-end
\ No newline at end of file
+
+  private def current_room
+    @current_room ||= current_workspace.rooms.friendly.accessable_by(current_person).find(params[:id])
+  end
```

Comment:
I'm not sure what `current_room` means in this case; but it looks kind of like we're starting to define `loader` methods; which makes me want to reach for a tool like [decent_exposure](https://github.com/hashrocket/decent_exposure)

Reply:
I was trying to expose the room show in `rooms/show.html.erb`.
Could be I haven't understand the benefit of [decent_exposure](https://github.com/hashrocket/decent_exposure) but don't feel like pull in a library to expose a room just yet.

Reply:
Yea, the value in decent_exposure is it creates a standard mechanism for exposing the data that a controller wants to expose in a way that is consistent across all controllers. I think it’s something to look into as we start wiring in pundit; or if we start defining helper methods to expose data to the UI instead of a single instance variable.